### DISCUSSION TITLE
Support for Web Assembly Hosting Model

### DISCUSSION THREAD

**Discussion by slamarreacosta** [MoonriseSoftwareCalifornia/AspNetCore.Identity.CosmosDb#15] [Ideas]
Hi,

I have a Blazor Server app that I would like to convert to a Web Assembly.

I am wondering if there are any plans to support Web Assembly for Asp Net Core Identity with CosmosDb.

If not, would it be possible to provide some code or details steps?

Thank you.

**Comment by toiyabe** [comment]
Hello @slamarreacosta,

I haven't planned on doing that. Can you perhaps provide me with an example of another identity provider that has been modified for compatibility?

Thanks!

**Comment by slamarreacosta** [comment]
Hi,

I do not have examples of another identity provider. It is just that I have a Blazor Server app that would beneficiate to be converted to a Web Assembly app and I am wondering if it was in your roadmap.

Are you planning to support .NET 8 with the Server hosting model?

Thanks.

**Reply by toiyabe** [reply]
Hi!
I haven't any plans to port this over to web assembly. A couple of weeks ago I have this package supporting.Net 8. By server hosting model what do you mean exactly? I may not be familiar with that.

**Reply by slamarreacosta** [reply]
Thank you for supporting .Net 8. Blazor is a Microsoft technology built on top of .NET. It is to build web sites. Check this link about the different hosting model for a Blazor application. You can also take Visual Studio 2022 and create a sample Blazor Web Assembly and look at the code. https://learn.microsoft.com/en-us/aspnet/core/blazor/hosting-models?view=aspnetcore-7.0

**Reply by etriebe** [reply]
@slamarreacosta Independent of web assembly, had you previously integrated any blazor components into the login/logout pages for this nuget package?

**Reply by slamarreacosta** [reply]
No. I changed my strategy. I went back to SQL Server to store all the identity data.
